### DISCUSSION TITLE
Unable to access parameters on 6.0

### DISCUSSION THREAD

**Discussion by caiochami** [Power-Components/livewire-powergrid#1816] [BUG]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

Yes - I didn't work.

### PHP Version

8.3.14

### PowerGrid

6.1.7

### Laravel

11.33.2

### Livewire

3.5.6

### Theme

Tailwind 3.x

### Describe the bug.

I'm no longer able to access component properties as parameters on export  as I used to on 5.0

![image](https://github.com/user-attachments/assets/dcc42201-ac5c-49ec-803b-25b097c88e9e)

**Comment by eafarooqi** [comment]
You can access the properties with `$this`

```
$this->teamId
```

**Comment by caiochami** [comment]
> You can access the properties with `$this`
> 
> ```
> $this->teamId
> ```

Hello. The property isn't available within the job when you export the data.

**Comment by eafarooqi** [comment]
yes, it has to be set in job. whats your logic behind this property?

**Comment by caiochami** [comment]
This is a SAAS application and the user should only see records that belong to their workspace. The teamId property is required to filter those records.

**Comment by luanfreitasdev** [comment]
Working normally in v6

![image](https://github.com/user-attachments/assets/cb1204f2-d37a-44ea-b79b-09d84a90ba6c)

**Comment by caiochami** [comment]
Hello @luanfreitasdev 

This isn't working for me. I'm currently using v6.1.10, and the error still persists.
![image](https://github.com/user-attachments/assets/f023860e-b49e-417d-b04d-8ee27c0f8cd6)
![image](https://github.com/user-attachments/assets/79f40362-61ae-464d-a18b-fed86ab3c6b4)
![image](https://github.com/user-attachments/assets/0003a404-e0fb-496e-95ad-e4dddf87192d)
![image](https://github.com/user-attachments/assets/f5fe2153-72e1-4d0c-8197-fbdf5cbaf79f)
![image](https://github.com/user-attachments/assets/8ec03eb0-d28f-4395-a9ca-88fd0339ae32)
![image](https://github.com/user-attachments/assets/8db7e8b5-0ef8-45c8-a217-64de551a2b61)

**Reply by luanfreitasdev** [reply]
https://github.com/Power-Components/livewire-powergrid/pull/2014
